pkgs/development/python-modules/biosppy/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+  buildPythonPackage,
+}:
+
+buildPythonPackage rec {
+  pname = "biosppy";
+  version = "2.2.2";
+
+  src = fetchFromGitHub {
+    owner = "scientisst";
+    repo = "BioSPPy";
+    tag = "v${version}";
+    hash = "sha256-U0ZftAlRlazSO66raH74o/6eP1RpmuFoA6HJ+xmgKR8=";
+  };
+
+  build-system = with python3Packages; [
+    bidict
+    h5py
+    matplotlib
+    numpy
+    scikit-learn
+    scipy
+    shortuuid
+    six
+    joblib
+    pywavelets
+    mock
+    sphinxHook
```

Comment:
Doc dependencies such as `sphinx` should generally be omitted unless the package needs them at runtime.